package.json

```diff
@@ -19,11 +19,11 @@
   "license": "ISC",
```

Comment:
Hey, 
Compilation failed due to missing changes in  package-lock.json
package-lock.json should be changed as well